05_syntax_lab/01.py

```diff
@@ -3,3 +3,14 @@
 the user and prints the largest one
 """
 
+largestNumber = int(0)
```

Comment:
If all inputs are negative then your program would print 0 as the largest value (and not for instance the actual largest value from the input).

One way around this is to differentiate the first value from the rest, and use the first read value as the initial value for `largestNumber`

A second approach is to use the lowest value in the world as the initial value. In python that lowest value is `float('-inf')` (which translates to the negative infinity or מינוס אינסוף)